www-apps/kibana-bin/Manifest

```diff
@@ -2,3 +2,4 @@ DIST kibana-5.6.8-linux-x86_64.tar.gz 51974963 BLAKE2B d90b3e56e9d19c3846b7fae29
 DIST kibana-6.0.1-linux-x86_64.tar.gz 62720465 BLAKE2B 81c3cb130067483cc6138c5274555c01102e365d873bf753a534cbcd96487516af41fbb91878cea4f80ffc02c5d18c82d322b0530cf1d3a55adffeefe60d9574 SHA512 3761f8abff5d949631f365c228af2f205597e1f05137d0140917b35128d2dcb79e32db355873864c1f01bce06121af71f1bfcb1679ae25903e1121bc76f68ecc
```

Comment:
Commit message nitpick: 'from' -> 'starting with'. 'From' works well with places or 'from ... to ...' formulas but not with single point of time.